modules/hrm/includes/class-announcement.php

```diff
@@ -196,6 +196,9 @@ function meta_boxes_cb( $post_id ) {
                         </select>
                     </td>
                 </tr>
+
+                <?php do_action( 'hr-announcement-sms-settings', $post ); ?>
```

Comment:
You can not add `sms` in the action name. Use something generic, like `hr_accouncement_table_last`